robocop/checkers/spacing.py

```diff
@@ -301,6 +317,38 @@ def validate_indent_lists(self, indents, previous_indent=None):
                             col=indent[0] + 1)
         return counter
 
+    def validate_standalone_comments(self, comments_and_eols):
```

Comment:
Please add some explanation in docstring for future generations 😅

Reply:
Who doesn't love magic methods? :D 

Added.